massa-execution-worker/src/common.rs

```diff
@@ -0,0 +1,48 @@
+use massa_ledger_exports::{LedgerController, SetOrDelete, SetUpdateOrDelete};
+use massa_models::address::Address;
+
+use std::collections::{BTreeSet, VecDeque};
+use massa_execution_exports::ExecutionOutput;
+
+/// Get every final and active datastore key of the given address
+pub fn get_final_and_candidate_datastore_keys(
```

Comment:
I recommend removing the "common" file as this function is only useful for the API (we don't need a fully copy of the final state in any other case)

Reply:
done